spec-tests/config/checkstyle/HEADER.txt

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
You don't need to duplicate the checkstyle config. If you rename the checkstyle tasks in spec-test build.gradle to `checkstyleSpec` as I mention in my other comment, the main config is picked up already, so you can delete the `spec-tests/config` folder